packages/devtools_app/lib/src/inspector/flutter/inspector_screen.dart

```diff
@@ -192,39 +192,32 @@ class _InspectorScreenBodyState extends State<InspectorScreenBody>
 
   Widget _expandCollapseButtons() {
     if (!_expandCollapseSupported) return null;
+
     return Align(
       alignment: Alignment.topRight,
-      child: Container(
-        padding: const EdgeInsets.symmetric(horizontal: 6),
-        // Add a semi-transparent background to the
```

Comment:
does this comment about avoiding interfering with the expand collapse buttons no longer apply?

Reply:
Yeah, I think that the buttons must have been overlaid over the content at one time? I don't think that's a bad idea, but currently they're above the content, on the same row as the tab buttons.

Reply:
oh now I remember! that was before there was the tab bar with the layout explorer.